update CSS to prevent event title wrapping

evlnyng opened the pull request:
## Description

Fixes #2299.

- Increased the bottom-margin for event icons to prevent long titles from wrapping beneath it.

## Further comments

This fix does increase the space between the events (pictured below; original on the left and my version on the right). 

<img width="601" alt="Screen Shot 2019-10-20 at 12 24 44 AM" src="https://user-images.githubusercontent.com/34118418/67243560-d8d02e00-f425-11e9-9c7e-3f683527e4ea.png">

If this is not preferred, please let me know and I will try a different solution.

## Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/master/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

@nasa-gibs/worldview

jasontk19 commented:
Nice!  I think this would work, but I bet we can prevent the wrap without adding additional vertical margin if we give the `h4` element some left margin to account for the icon:
![Screen Shot 2019-10-22 at 9 20 33 AM](https://user-images.githubusercontent.com/1480000/67289929-63a83b80-f4ad-11e9-962b-d5a284a56a51.png)
![Screen Shot 2019-10-22 at 9 19 48 AM](https://user-images.githubusercontent.com/1480000/67289930-63a83b80-f4ad-11e9-9a25-6b38903c4b19.png)


Also, it looks like you're working off of your own forked version of our repo.  If you can, go ahead and clone our repo directly and create your branch there.  This way other developers won't have to clone your fork to checkout your branches to test them.  Let me know if that doesn't make sense or you want some help with it!

evlnyng commented:
Understood! I can absolutely do that, thank you. I will submit a new request with the new branch.

jasontk19 commented:
@evlnyng I was mistaken; it turns out you need to have permissions to push a branch to this repository directly.  You can open the PR off of your forked branch like you did originally!

evlnyng commented:
I thought that might've been the case! I have reverted the changes to the event icon and added the left margin to the `h4` element. Thank you for your patience!

jasontk19 commented on web/css/events.css:
```diff
@@ -24,6 +24,7 @@
   font-size: 0.85em;
   color: #ccc;
   padding: 0 3px;
+  left-margin: 40px; 
```
Almost there! The proper syntax here is going to be `margin-left`

evlnyng replied:
How embarassing!. 🤦‍♀️ Serves me right for trying to rush it during a break at work.

jasontk19 replied:
No worries!  I think we all do the same sort of thing pretty much daily 👍 

Thanks for your contribution!
